controllers/controllers/workloads/cfspace_controller.go

```diff
@@ -324,14 +325,22 @@ func (r *CFSpaceReconciler) finalize(ctx context.Context, log logr.Logger, space
 		return ctrl.Result{}, nil
 	}
 
-	err := r.finalizeCFApp(ctx, log, space.GetName())
-	if err != nil {
-		log.Error(err, "failed to finalize CFApps while deleting CFSpace")
+	duration := time.Since(space.GetDeletionTimestamp().Time)
+	log.Info(fmt.Sprintf("finalizing CFSpace for %fs", duration.Seconds()))
+	if duration < 60.0*time.Second {
+		err := r.finalizeCFApps(ctx, log, space.GetName())
+		if err != nil {
+			log.Error(err, "failed to finalize CFApps while deleting CFSpace")
```

Comment:
The way the logic now looks, this isn't really failure anymore, right? We just haven't finished finalizing yet. So this should probably also be a log.Info?

Reply:
That's true. Fixed.